mysite/account/tests.py

```diff
@@ -15,38 +15,41 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+import logging
 import os
-import tempfile
 import StringIO
-import logging
+import tempfile
 from webtest import Upload
 
-from mysite.profile.models import Person
-import mysite.account.forms
-from mysite.search.models import Project, ProjectInvolvementQuestion
-import mysite.project.views
-import mysite.account.views
-
 from django.contrib.auth.models import User
 from django.contrib.auth import authenticate
-from django.test.client import Client
 from django.core.urlresolvers import reverse
+from django.test.client import Client
 from django.utils.unittest import skipIf
-
 from django_webtest import WebTest
+
+import mysite.account.forms
+import mysite.account.views
 import mysite.base.depends
+from mysite.profile.models import Person
+import mysite.project.views
+from mysite.search.models import Project, ProjectInvolvementQuestion
 
 
 class Login(WebTest):
+    """ Tests for user logins and logouts """
     fixtures = ['user-paulproteus', 'person-paulproteus']
 
     def test_login(self):
+        """ Test login with username and password """
         user = authenticate(username='paulproteus',
                             password="paulproteus's unbreakable password")
-        self.assert_(user and user.is_active)
```

Comment:
@ehashman Great question. See line 47. `assert_` was deprecated in Python 2.7 and replaced with `assertTrue`. https://docs.python.org/2/library/unittest.html?highlight=unittest#deprecated-aliases  I was just going to the more common (now) usage.